lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -2341,14 +2341,21 @@ LogicalResult FIRRTLLowering::visitDecl(WireOp op) {
     symName = builder.getStringAttr(moduleNamespace.newName(
         Twine("__") + moduleName + Twine("__") + name.getValue()));
   }
+
+  bool hasChiselName = false;
```

Comment:
perhaps instead of `ChiselName` could be something like `hasUserSpecifiedName` ?

Reply:
Thanks, I agree! I change to `userSpecifiedName`.